docker-bits/5_DB-Drivers.Dockerfile

```
@@ -2,4 +2,6 @@
 RUN apt-get update && \
     apt-get install -y alien unixodbc unixodbc-dev && \
     pip install --no-cache-dir --quiet 'pyodbc==4.0.30' && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    fix-permissions $CONDA_DIR && \
+    fix-permissions /home/$NB_USER
```

End files with empty line
```suggestion
    fix-permissions /home/$NB_USER

```